Cogs.Publishers/CsSchemaPublisher.cs

```diff
@@ -644,7 +688,9 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                 var props = obj.Children();
                 if (single != null)
                 {
-                    var id = props.ElementAt(1).First.Last.ToString();
+                    string id = null;
+                    try { id = props.ElementAt(0).First.Last.ToString(); }
+                    catch (Exception) { id = props.ElementAt(1).First.Last.ToString(); }
```

Comment:
Exception catching shouldn't be used as control flow for known paths. Is there a way to look at the data first before calling first and last?